internal/routing/local.go

```diff
@@ -85,3 +90,14 @@ func (r *Routing) LocalNetworks() (localNetworks []LocalNetwork, err error) {
 
 	return localNetworks, nil
 }
+
+func (r *Routing) AddLocalRules(subnets []LocalNetwork) (err error) {
+	for _, net := range subnets {
+		// Main table was setup correctly by Docker, just need to add rules to use it
+		err = r.addIPRule(nil, net.IPNet, mainTable, localPriority)
```

Comment:
nit remove the global constants and inline them here since they are only used here.

```go
		const mainTable = 254
		const localPriority = 98
		err = r.addIPRule(nil, net.IPNet, mainTable, localPriority)
```

Reply:
Thanks for the feedback. Updated as requested.